app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file = parts[1]
+    dest_file = parts[2]
```

Comment:
```suggestion
    _, source_file, dest_file = parts
```
better use unpacking in this case